mbed_lpc1768 doesn't build

With `examples/default` this fails:

 `make BOARD=mbed_lpc1768 -B clean all`

```
/foo/RIOT/examples/default/bin/mbed_lpc1768/core.a(hwtimer.o): In function `hwtimer_spin':
/foo/RIOT/core/hwtimer.c:65: undefined reference to `hwtimer_arch_now'
/foo/RIOT/core/hwtimer.c:75: undefined reference to `hwtimer_arch_now'
/foo/RIOT/core/hwtimer.c:86: undefined reference to `hwtimer_arch_now'
/foo/RIOT/examples/default/bin/mbed_lpc1768/core.a(hwtimer.o): In function `hwtimer_init_comp':
/foo/RIOT/core/hwtimer.c:100: undefined reference to `hwtimer_arch_init'
/foo/RIOT/examples/default/bin/mbed_lpc1768/core.a(hwtimer.o): In function `hwtimer_now':
/foo/RIOT/core/hwtimer.c:120: undefined reference to `hwtimer_arch_now'
/foo/RIOT/examples/default/bin/mbed_lpc1768/core.a(hwtimer.o): In function `_hwtimer_set':
/foo/RIOT/core/hwtimer.c:177: undefined reference to `hwtimer_arch_set_absolute'
/foo/RIOT/core/hwtimer.c:181: undefined reference to `hwtimer_arch_set'
/foo/RIOT/examples/default/bin/mbed_lpc1768/core.a(hwtimer.o): In function `hwtimer_remove':
/foo/RIOT/core/hwtimer.c:209: undefined reference to `hwtimer_arch_disable_interrupt'
/foo/RIOT/core/hwtimer.c:210: undefined reference to `hwtimer_arch_unset'
/foo/RIOT/core/hwtimer.c:217: undefined reference to `hwtimer_arch_enable_interrupt'
collect2: error: ld returned 1 exit status
/foo/RIOT/examples/default/../../Makefile.include:61: recipe for target 'all' failed
make: *** [all] Error 1
```

--- a/boards/mbed_lpc1768/hwtimer_board.c
+++ b/boards/mbed_lpc1768/hwtimer_board.c
@@ -1,0 +1,67 @@
+/**
+ * MBED LPC1768 board hwtimer_arch.h implementation stub
+ *
+ * TODO: implement
+ *
+ * Copyright (C) 2014 Ludwig Ortmann
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License. See the file LICENSE in the top level directory for more
+ * details.
+ *
+ * @ingroup hwtimer
+ * @ingroup mbed_lpc1768
+ * @{
+ * @author  Ludwig Ortmann <ludwig.ortmann@fu-berlin.de>
+ * @file
+ * @}
+ */
+
+#include <inttypes.h>
+
+#include "hwtimer_arch.h"
+
+#define ENABLE_DEBUG (1)
+#include "debug.h"
+
+void hwtimer_arch_enable_interrupt(void)
+{
+    DEBUG("hwtimer_arch_enable_interrupt(): not implemented\n");
+    return;
+}
+
+void hwtimer_arch_disable_interrupt(void)
+{
+    DEBUG("hwtimer_arch_disable_interrupt(): not implemented\n");
+    return;
+}
+
+void hwtimer_arch_unset(short timer)
+{
+    DEBUG("hwtimer_arch_unset(%i): not implemented\n", timer);
+    return;
+}
+
+void hwtimer_arch_set(unsigned long offset, short timer)
+{
+    DEBUG("hwtimer_arch_set(%lu, %i): not implemented\n", offset, timer);
+    return;
+}
+
+void hwtimer_arch_set_absolute(unsigned long value, short timer)
+{
+    DEBUG("hwtimer_arch_set_absolute(%lu, %i): not implemented\n", value, timer);
+    return;
+}
+
+unsigned long hwtimer_arch_now(void)
+{
+    DEBUG("hwtimer_arch_now(): not implemented\n");
+    return 0;
+}
+
+void hwtimer_arch_init(void (*handler)(int), uint32_t fcpu)
+{
+    DEBUG("hwtimer_arch_init(%p, %"PRIu32"): not implemented\n", handler, fcpu);
+    return;
+}

